assets/src/edit-story/components/canvas/selectionCanvas.js

```diff
@@ -24,6 +24,11 @@ const LassoMode = {
 };
 
 const Container = withOverlay( styled.div`
+	position: absolute;
```

Comment:
nit: fix indentation

Reply:
hmm. the `--fix` does some rather strange things...

Reply:
Might be improved a bit with the new repo, but linting and autofixing styled components is tricky